scalding-core/src/main/scala/com/twitter/scalding/Job.scala

```diff
@@ -77,6 +77,7 @@ object Job {
  * these functions can be combined Monadically using algebird.monad.Reader.
  */
 class Job(val args: Args) extends FieldConversions with java.io.Serializable {
+  Tracing.init()
```

Comment:
A document service is a plugin deployed at runtime. The programmer may not have any idea that one is deployed.

The tracing code change is just registering a regex representing the boundary of the internals of Scalding. It runs once at start up and has no impact on users.